src/rmem.h

```diff
@@ -378,11 +378,11 @@ MemPool CreateMemPool(const size_t size)
 {
     MemPool mempool = { 0 };
 
-    if (size == 0) return mempool;
+    if ((size == 0) || (size < sizeof(MemNode))) return mempool;
     else
     {
         // Align the mempool size to at least the size of an alloc node.
-        uint8_t *const restrict buf = malloc(size*sizeof *buf);
+        uint8_t *const restrict buf = malloc(size*sizeof(uint8_t));
 
```

Comment:
The comment above claims that it should be aligned to sizeof(MemAlloc) but I don't see it implemented.